asctb-api/src/functions/api.functions.ts

```diff
@@ -71,9 +71,16 @@ function setData(column: string[], row: any, value: any, warnings: Set<string>):
  * @param warnings = warnings generated during the process are pushed to this set
  * @returns = returns key value pairs of metadata
  */
-const buildMetadata = (metadataRows: string[][], warnings: Set<string>): Record<string, string> => {
+const buildMetadata = (metadataRows: string[][], warnings: Set<string>): Record<string, string | string[]> => {
+  const [titleRow] = metadataRows.splice(TITLE_ROW_INDEX, 1);
+  const [title] = titleRow;
```

Comment:
A bit brittle if they happen to have some extra columns this will end up being an array. Can probably just do titleRow.slice(0, 1).

Reply:
added slice(0, 1) to totalRow